php/php.editor/test/unit/data/testfiles/actions/testNB4978/01/testNB4978_01.php.fixUses

```diff
@@ -23,8 +23,6 @@ declare (strict_types=1);
 
 namespace TestNamespace;
 
-use Foo\Foo\Foo;
```

Comment:
Why?

Reply:
> To have Fix Imports working for the whole file I had to add scope to UsedNamesCollector's names and it also affected some tests, because for now it is important that DataItems from ImportDataCreator were paired with Selections, but previously Selection could be imported without real DataItem.

We fixed this once, so, it would be recognized as a regression, I think...

Reply:
It because there is no one real namespace in file or near to it, so if you debug UsesNamespaceCollector all dataItems names will be empty. So if I remove check for UsedNames you could put to selection anything like "Just\Want\It\To\Be\Added\To\Uses" and it would be added. For this test it would work as previously if we put file with \Foo\Foo\Foo near to the test file.

Reply:
I've added class file to have tests in previous state and don't confuse anyone.